Spigot-Server-Patches/0182-Update-VehicleCreateEvent.patch

```diff
@@ -0,0 +1,124 @@
+From fa34964d7a14b0ec6f6c1522c747ef6e438b1c81 Mon Sep 17 00:00:00 2001
+From: AlphaBlend <whizkid3000@hotmail.com>
+Date: Sun, 6 Nov 2016 12:38:31 -0800
+Subject: [PATCH] Update VehicleCreateEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/DispenserRegistry.java b/src/main/java/net/minecraft/server/DispenserRegistry.java
+index 41bd6e5..2bb66bb 100644
+--- a/src/main/java/net/minecraft/server/DispenserRegistry.java
++++ b/src/main/java/net/minecraft/server/DispenserRegistry.java
+@@ -759,6 +759,13 @@ public class DispenserRegistry {
+             EntityBoat entityboat = new EntityBoat(world, event.getVelocity().getX(), event.getVelocity().getY(), event.getVelocity().getZ());
+             // CraftBukkit end
+ 
++            // Paper start
```

Comment:
Maybe move this down just before the world.addEntity(), so all properties could be used by event handlers.